src/components/gamePlayConsole/GamePlayConsole.tsx

```diff
@@ -30,10 +34,51 @@ const generateComic = (img: string) => {
     );
 };
 
+const generateMiniGame = (minigame: IMinigame, finishMinigame: Function) => {
+    let unityBuildName: string = minigame.build;
+    let unityContent = new UnityContent(
+        `unity/${unityBuildName}/Build/${unityBuildName}.json`,
+        `unity/${unityBuildName}/Build/UnityLoader.js`,
+    );
+
+    unityContent.on("WinMiniGame", () => {
```

Comment:
May need to document somewhere that we have 2 triggers that must trigger when developing minigames.